### DISCUSSION TITLE
Event not sending email to participants

### DISCUSSION THREAD

**Discussion by Bryntet** [kuzmoyev/google-calendar-simple-api#164] [General]
As the title says, emails are not being sent out as they would with normal google calendar event creations.

**Comment by kuzmoyev** [comment]
To send notifications, you need to use `send_updates` parameter of the `add_event`, `update_event`, etc.:

```python
from gcsa.google_calendar import SendUpdatesMode

...

gc.add_event(event, send_updates=SendUpdatesMode.ALL)
```

See more info in the [docs of `SendUpdatesMode`](https://google-calendar-simple-api.readthedocs.io/en/latest/code/google_calendar.html#gcsa.google_calendar.SendUpdatesMode)

**Comment by Bryntet** [comment]
Thank you so much!!
